hudi-utilities/src/test/java/org/apache/hudi/utilities/TestHiveIncrementalPuller.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.hudi.utilities;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestHiveIncrementalPuller {
+
+  private HiveIncrementalPuller.Config config;
+
+  @Before
+  public void setup() {
+    config = new HiveIncrementalPuller.Config();
+  }
+
+  @Test
+  public void testInitHiveIncrementalPuller() throws Exception {
```

Comment:
nit: do we still need the `throws Exception`? :)

Reply:
Good catch 👍 . Sorry, I forgot to delete it.